application/forms/Command/CommandForm.php

```diff
@@ -23,6 +23,15 @@ abstract class CommandForm extends Form
     /** @var mixed */
     protected $objects;
 
+    /**
+     * Whether an error occurred while sending the command and used to
+     *
+     * prevent the success message from being rendered simultaneously
```

Comment:
The first line should be a complete sentence. Any additional information can of course be mentioned in the extended description, but the main description has to be self-sufficient.